admin-site/src/components/HelloWorld.vue

```diff
@@ -3,13 +3,17 @@
   <img src="C:\Users\amydo\Downloads\login.jpg" alt="Login image">-->
   <div class="login">
     <div style="display: flex; justify-content: center; align-items: center; height: 10vh;">
-  <h1>Pluggable Login</h1>
+  <h1>Admin-site Login</h1>
     </div>
     <form>
       <div class="form-group">
```

Comment:
I see you're using pure html elements here my advice would be to use the vuetify components, they have a lot of extra feature such as validation etc and will make the site look consistent throughout as they all use the same design system

Here's an example of a login form using the vuetify components https://vuetifyjs.com/en/components/text-fields/#forms